plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/S3HudiQueryRunner.java

```diff
@@ -152,7 +159,7 @@ public static void main(String[] args)
         Logging.initialize();
         Logger log = Logger.get(S3HudiQueryRunner.class);
 
-        String bucketName = "test-bucket";
+        String bucketName = "test-bucket-" + randomTableSuffix();
```

Comment:
Why `randomTableSuffix` is needed?

Reply:
Just in case older bucket is not cleaned up. So, with `randomTableSuffix` it always starts with a different bucket name.

Reply:
I don't think it happens because we don't reuse the container. Please remove it.